### DISCUSSION TITLE
Neovim plugin does not accept suggestion

### DISCUSSION THREAD

**Discussion by elefthei** [community/community#136878] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have Neovim + Plug + CoC + Copilot. I can see the copilot suggestions but cannot accept them with <TAB>. I suspect CoC is remaping TAB for its own autocomplete. I would like to remap copilot#Accept() to Shift-Tab but the obvious thing is not working

`imap <silent> <S-TAB> copilot#Accept()`

How can I remap accept so it works?

**Comment by mrjosh** [comment, marked as answer]
To remap `copilot#Accept()` to `Shift-Tab` in Neovim, 
you can add a keybinding in your Neovim configuration file (`init.vim` or `init.lua`). 

Here's how to do it for both `init.vim` and `init.lua`:
***You might want to add this configurations to the very bottom of your init file***

### init.vim
```lua
let g:copilot_no_tab_map = v:true
inoremap <expr> <S-Tab> copilot#Accept("<CR>")
```

### init.lua
```lua
vim.g.copilot_no_tab_map = true
vim.api.nvim_set_keymap('i', '<S-Tab>', 'copilot#Accept("<CR>")', { noremap = true, silent = true, expr = true })
```

**Reply by elefthei** [reply]
Thanks that did it!
